source/dub/packagemanager.d

```diff
@@ -624,128 +661,30 @@ class PackageManager {
 		foreach (p; this.completeSearchPath)
 			scanPackageFolder(p);
 
-		void loadOverrides(LocalPackageType type)
+		foreach (ref repo; m_repos)
 		{
-			m_repositories[type].overrides = null;
-			auto ovrfilepath = m_repositories[type].packagePath ~ LocalOverridesFilename;
+			repo.overrides = null;
+			auto ovrfilepath = repo.packagePath ~ LocalOverridesFilename;
 			if (existsFile(ovrfilepath)) {
 				foreach (entry; jsonFromFile(ovrfilepath)) {
 					PackageOverride ovr;
 					ovr.package_ = entry["name"].get!string;
 					ovr.version_ = Dependency(entry["version"].get!string);
 					if (auto pv = "targetVersion" in entry) ovr.targetVersion = Version(pv.get!string);
 					if (auto pv = "targetPath" in entry) ovr.targetPath = Path(pv.get!string);
-					m_repositories[type].overrides ~= ovr;
+					repo.overrides ~= ovr;
 				}
 			}
 		}
-		loadOverrides(LocalPackageType.user);
-		loadOverrides(LocalPackageType.system);
 	}
 
-	alias Hash = ubyte[];
+	deprecated alias Hash = ubyte[];
 	/// Generates a hash value for a given package.
 	/// Some files or folders are ignored during the generation (like .dub and
 	/// .svn folders)
-	Hash hashPackage(Package pack)
```

Comment:
seemed strange to have this here, seeing as it only applies to a single package, so it's now a member of `Package`. Kept a `deprecated` shim here for compatibility seeing as it was `public`.